static/app/views/dashboards/manage/dashboardTable.tsx

```diff
@@ -177,6 +220,8 @@ function DashboardTable({
   return (
     <GridEditable
       data={dashboards ?? []}
+      // necessary for edit access dropdown
+      bodyStyle={{overflow: 'inherit'}}
```

Comment:
I don't think `inherit` is super explicit because it depends on its parent. Can we just give it `overflow: visible`?